app/lib/actions/katello/repository/sync.rb

```diff
@@ -75,7 +79,7 @@ def humanized_name
         end
 
         def presenter
-          Helpers::Presenter::Delegated.new(self, planned_actions(Pulp::Repository::Sync))
+          Helpers::Presenter::Delegated.new(self, planned_actions(Pulp::Orchestration::Repository::Sync))
```

Comment:
I changed this method ot just: 

```
        def presenter
          found = all_planned_actions(Pulp::Repository::Sync)
          found = all_planned_actions(Pulp3::Repository::Sync) if found.empty?
          Helpers::Presenter::Delegated.new(self, found)
        end
```
and it seemed to work great:

![image](https://user-images.githubusercontent.com/395077/56255523-407e7880-6093-11e9-85ea-35b7bc54c71b.png)

Reply:
and then you can remove the presenter logic from the orchestration, the pulp selector, and controller

Reply:
Aah..This works perfect! Wonder what I was trying to do..I will make the changes..